pr_watch/tests/test_models.py

```diff
@@ -141,7 +144,10 @@ def test_create_from_pr(self):
         self.assertEqual(watched_pr.fork_name, pr.fork_name)
         self.assertEqual(watched_pr.branch_name, pr.branch_name)
 
-        self.assertEqual(instance.sub_domain, 'pr{}.sandbox'.format(pr.number))
+        internal_lms_domain = 'pr{}.sandbox.basedomain.com'.format(pr.number)
+        self.assertEqual(instance.internal_lms_domain, internal_lms_domain)
+        self.assertEqual(instance.internal_lms_preview_domain, 'preview-{}'.format(internal_lms_domain))
+        self.assertEqual(instance.internal_studio_domain, 'studio-{}'.format(internal_lms_domain))
```

Comment:
@mtyaka Instead of hardcoding default prefixes (`preview-`, `studio-`) here, use values from settings? That way this test would also ensure that `settings.DEFAULT_STUDIO_DOMAIN_PREFIX` and `settings.DEFAULT_LMS_PREVIEW_DOMAIN_PREFIX` are respected. (It might also make sense to keep the hardcoded prefixes here and adjust one of the other tests instead; just mentioning this now because I didn't notice it earlier. Overall I think it would be worth having at least one test that checks if prefixes are equal to corresponding settings.)